modules/services/way-displays.nix

```diff
@@ -79,14 +86,20 @@ in
       (lib.hm.assertions.assertPlatform "services.way-displays" pkgs lib.platforms.linux)
     ];
 
-    xdg.configFile."way-displays/cfg.yaml".source =
-      yaml.generate "way-displays-config.yaml"
-        (mergeSets [
-          {
-            CALLBACK_CMD = "${pkgs.libnotify}/bin/notify-send \"way-displays \${CALLBACK_LEVEL}\" \"\${CALLBACK_MSG}\"";
-          }
-          cfg.settings
-        ]);
+    home.packages = [ cfg.package ];
+
+    wayland.windowManager = lib.mapAttrs (name: _: {
+      systemd.variables = [ "XDG_VTNR" ];
+    }) options.wayland.windowManager;
+
+    xdg.configFile."way-displays/cfg.yaml" = mkIf (cfg.settings != null) {
+      source = yaml.generate "way-displays-config.yaml" (mergeSets [
+        {
+          CALLBACK_CMD = "${pkgs.libnotify}/bin/notify-send \"way-displays \${CALLBACK_LEVEL}\" \"\${CALLBACK_MSG}\"";
```

Comment:
Losing this sucks. It's like... there should be both a configuration file and a state file.

Reply:
that's a request for way-displays I guess